src/env.js

```diff
@@ -24,4 +24,25 @@ module.exports = envalid.cleanEnv(process.env, {
     default: 4001,
     desc: 'subscription port (websocket)',
   }),
+  PAYPAL_MODE: envalid.str({
+    default: 'sandbox',
+    desc: 'paypal environment sandbox | live',
+  }),
+  PAYPAL_CLIENT_ID: envalid.str({
+    default: 'NULL',
```

Comment:
You can just remove this line and the same one below. NULL as string is a valid number otherwise.

Reply:
I made this on purpose.